src/cr/cube/matrix.py

```diff
@@ -1027,6 +1093,10 @@ def means(self):
     def numeric(self):
         return self._base_vector.numeric
 
+    @lazyproperty
+    def opposite_margins(self):
+        return self._base_vector._opposite_margins
```

Comment:
These should  be made public on the base vector class... we shouldn't access private properties.